app/views/status_updates/show.html.slim

```diff
@@ -12,6 +12,30 @@ p
 .well.status_update
   = render_markdown(@status_update.content).html_safe
 
+h2#comments.header
+  = icon('comments')
+  span Comments
+
+- if current_user
+  = simple_form_for Comment.new(commentable: @status_update), url: comments_path do |f|
```

Comment:
That is the reason why you don't see any effect from the new controller: this is still posting to the old one. You could use something like: 'url: status_updates_comments_path'.